patches/server/0062-Add-methods-for-working-with-arrows-stuck-in-living-.patch

```diff
@@ -6,11 +6,32 @@ Subject: [PATCH] Add methods for working with arrows stuck in living entities
 Upstream added methods for this, original methods are now
 deprecated
 
+diff --git a/src/main/java/net/minecraft/world/entity/LivingEntity.java b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+index 3eeb40c2176a80b9e2a472d43671ae0fe087d9e7..370ba23256cfbfe591c13c9052cef25ee74b6974 100644
+--- a/src/main/java/net/minecraft/world/entity/LivingEntity.java
++++ b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+@@ -188,6 +188,7 @@ public abstract class LivingEntity extends Entity implements Attackable {
+     public InteractionHand swingingArm;
+     public int swingTime;
+     public int removeArrowTime;
++    public int maximumArrowsInBody; // Paper - Add methods for working with arrows stuck in living entities
```

Comment:
Isn't it necessary to define a vanilla-default value here (`30`)?
Because now, if I understood correctly, it will be `0` and this PR will prevent having arrows in body by default at all...
Same for stingers

Reply:
woops forgot to set that lol